sync2/dbset/dbset.go

```diff
@@ -92,7 +93,7 @@ func (d *DBSet) Received() rangesync.SeqResult {
 			}
 		},
 		Error: rangesync.NoSeqError,
-	}
+	}, len(d.received)
```

Comment:
It doesn't fully defeat the purpose as using iterators is preferable to creating temporary slices which substantially increase GC pressure and thus incur not-immediately-obvious CPU costs.
Yet, in this case we only actually need to know whether the returned sequence is not empty and this can be done by checking if it contains any elements. Will fix

Reply:
Fixed, sequence length is no longer returned